templates/sshd_config.erb

```diff
@@ -18,8 +18,8 @@ Port <%= @sshd_config_port %>
 #Protocol 2,1
 Protocol 2
 #AddressFamily any
```

Comment:
Might be nice to include the default commented line ('#AddressFamily any') in the if condition as well, since the option is not supported on Solaris. That avoids confusion for anyone going through the sshd_config file.